x-pack/plugins/security_solution/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -388,6 +394,7 @@ export const getResult = (): RuleAlertType => ({
         ],
       },
     ],
+    timestampOverride: undefined,
     references: ['http://www.example.com', 'https://ww.example.com'],
     note: '# Investigative notes',
     version: 1,
```

Comment:
Doesn't have to be this PR, but we should move towards getting more code out of __mocks__ and as close to their respective schema files where possible or just move them into a closely named file_name.mock.ts somewhere.

Just keep it in mind as you're going through here that I think it's more Kibana core like to move things out of the __mocks__ folders and it has just been easier to maintain and find mocks that way IMHO.